regulations/make.py

```diff
@@ -210,7 +211,12 @@ def buildLanguage(args, language):
 
   pdfName = languageData[language]["pdf"]
 
-  html.html(language, buildDir, pdfName + ".pdf", isTranslation=isTranslation, verbose=args.verbose)
+  srcDir = "translations/" + language if isTranslation else "wca-regulations"
+
+  warnings = check_missing.check_guidelines(language, srcDir)
+  if len(warnings) > 0:
+    print warnings.encode('utf-8')
```

Comment:
Sure, here is a toy example : 

test.py
```python
# coding=utf-8
import sys
a = "é"
print sys.stdout.encoding
print sys.getdefaultencoding()
print type(a)
print a
a = a.decode("utf-8")
print type(a)
print a
```


```
$ python test.py 
UTF-8
ascii
<type 'str'>
é
<type 'unicode'>
é
```
```
$ python test.py > toto 
Traceback (most recent call last):
  File "test.py", line 10, in <module>
    print a
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 0: ordinal not in range(128)
```
```
$ cat toto 
None
ascii
<type 'str'>
é
<type 'unicode'>
```

Notice how stdout encoding went from "utf-8" to none, leading python to using ascii to decode the unicode a !